great_expectations/expectations/expectation.py

```diff
@@ -861,6 +893,8 @@ def validate(
             configurations=[configuration],
             runtime_configuration=runtime_configuration,
         )[0]
+        if include_rendered_content:
+            evr.render()
```

Comment:
This invokes the `InlineRenderer` after the instance attributes `rendered_content` have been added and evaluation parameters have been processed.